.gitmodules

```diff
@@ -10,3 +10,7 @@
 	path = polkadot-sdk
 	url = git@github.com:paritytech/polkadot-sdk.git
 	branch = donal-ahm
+[submodule "zombie-bite/polkadot-sdk-doppelganger"]
+	path = zombie-bite/polkadot-sdk-doppelganger
+	url = git@github.com:paritytech/polkadot-sdk.git
```

Comment:
Why is it using git to clone? Cloning public repos can be done over https.

Reply:
Ahh yes, sorry. I just used the `ssh` url when I added the submodule.  I will change it.